angularSTEP/src/app/recipe-page/recipe-page.component.ts

```diff
@@ -115,4 +123,52 @@ export class RecipePageComponent {
       }
     });
   }
+
+  isRecipeInWishlist() {
+    if (this.user && this.loggedIn && this.id) {
```

Comment:
It's interesting that both `user` and `loggedIn` are tested here, since it seems like `loggedIn` is true iff `user` is truthy. Is `loggedIn` even needed at all if testing for `user` is possible and suffices?